### ISSUE TITLE
Set unit models based on Vic2 culture

### ISSUE THREAD

**Issue by Idhrendur** [Idhrendur/paradoxGameConverters#343]
https://forum.paradoxplaza.com/forum/index.php?threads/the-vic2-to-hoi4-converter-project-development-thread.915440/page-47#post-23233126
"I have an idea for units models : try to clone the existing countries based on the primary culture of vic2, for example prussia should clone the german units. I know how to do it in the files and i've done some C++ if you need help. Maybe you can add some generic units as well based on mods ? Why not ancient romanian units for the generic eastern graphical culture for example ? "

https://forum.paradoxplaza.com/forum/index.php?threads/the-vic2-to-hoi4-converter-project-development-thread.915440/page-48#post-23234342
"You have to add a file in gfx/entities/ and then create a file yourFileName.asset and copy this :

#################################
#################################
entity = {
   clone = "FRA_infantry_entity"
   name = "BEL_infantry_entity"
}
entity = {
   clone = "FRA_infantry_2_entity"
   name = "BEL_infantry_2_entity"
}
#################################

following this example this can lead to :

entity = {
   clone = "clonedCountryTagHere_infantry_entity"
   name = "yourTagHere_infantry_entity"
}
entity = {
   clone = "clonedCountryTagHere_infantry_2_entity"
   name = "yourTagHere_infantry_2_entity"
}

i assume it should work the same for tanks and planes. I already try this for a previous converted save and it have worked."

**Comment by derHochmeister** [comment]
I disagree with this for a simple motivation. Those unit skins were specifically for those specific nations of OTL. The probability North Germany would have still used SS-styled uniforms in other timelines is pretty rare, considering cases where fascism never entered Germany at all.

Unit-voice mappings, instead, that I already proposed, have a degree of plausibility, since they're based upon "culture" (which roughly is language).
